src/applications/avs/components/MedicationTerms.jsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+const MedicationTerms = props => {
+  const { avs } = props;
+
+  if (!avs.pharmacyTerms?.length) {
+    return null;
```

Comment:
Are you sure? Wouldn't a message saying something like "you don't have any medications" be better? I can't see the entire page from just the screenshots to know if there is anything else there.

Reply:
@kristenmcconnell @ninaranusavice  any thoughts on this? I'm assuming that we wouldn't show the widget at all if there are no terms to define, but that could have been a bad assumption 🙂